app/posts/digital-public-works-ideas.md

```diff
@@ -0,0 +1,86 @@
+
+I’m also big supporter of these. This unofficial X-GOVUK community is one way to support cross-government collaboration and tools, but many digital public works would need full time teams to support them.
+
+There’s lots to be thankful for. The [platforms developed by the Government Digital Service](https://www.gov.uk/service-toolkit#platforms-and-tools) – Notify, Pay, One Login and Forms – are all super useful.
+
+Unfortunately there doesn’t currently seem to be an easy way for government departments working on services to spot other opportunities for common platforms and to turn these into funded cross-government teams and services.
+
+However, were this to be possible, I do have a list of ideas for what these services could be.
+
+## The ideas
+
+### 1. Analytics
+
+There are plenty of existing commercial and open source tools for this, so I’m not suggesting rebuilding those, but instead picking one and building a service on top of it. 
+
+The team would do the hard work of navigating the trade-offs between privacy, security and usability to give teams the design patterns and technical implementation for collecting consent and then data.
+
+They might also build a simple web service to help teams view and understand the most important metrics, with comparisons across services.
+
+### 2. Address lookup
+
+The [public sector geospatial agreement](https://www.ordnancesurvey.co.uk/customers/public-sector/public-sector-geospatial-agreement) means that all public sector services should have access to high quality databases of UK addresses. However that doesn’t mean they are easy to integrate with, and some services still end up using Google or other commercial services.
+
+A central digital team could help by maintaining an API, design patterns and code to make it super easy for teams to let users lookup and select addresses from postcodes.
+
+### 3. Hosting
+
+This might have been tried once before, with [PaaS then being decommissioned](https://gds.blog.gov.uk/2022/07/12/why-weve-decided-to-decommission-gov-uk-paas-platform-as-a-service/), but it’s still a good idea.
+
+A rebooted service could be simpler but more opinionated, with the benefit of taking care of more of the low-level issues like operating systems, backups, monitoring and scaling.
+
+It could also support easy hosting of prototypes.
+
+### 4. Certifications
+
+There are lots of services where the outcome is that the user receives a certificate or qualification to do something, whether that’s teaching children in schools or working on gas boilers.
+
+Currently, each of these services needs to devise their own way for users to be able to prove or check these certifications.
+
+A central service could allow users to collect all their certifications on a single account and selectively share these with others, either via a code or some other mechanism.
+
+### 5. Registers
```

Comment:
I’m pretty sure there’s legislation that says this sort of thing couldn’t be built and maintained centrally. It’s also a massive risk to privacy, security and politics (dictators love central databases of people). That’s why models of data exchange for identity platforms look at decentralised storage and re-use, sharing verified credentials or facts rather than having one big database. 

So the technology for this sort of thing is likely to be built into One Login rather than be its own platform.

Reply:
I've tweaked the paragraph to make it clearer it's about public registers, rather than ones where the register is held by government but not published.